lib/kb_Metrics/metricsDBs.py

```diff
@@ -267,8 +272,12 @@ def get_user_info(self, userIds, minTime, maxTime, exclude_kbstaff=False):
     # Begin functions to query the other dbs...
     def aggr_activities_from_wsobjs(self, minTime, maxTime):
         # Define the pipeline operations
+        epoch = datetime.datetime.utcfromtimestamp(0)
+        datetime_moddate = {"$project": {"moddate": {"$add": [ epoch, "$moddate"]},
+                            "name": 1, "id": 1, "numver": 1, "ws": 1, "_id": 0}},
         pipeline = [
-            {"$match": {"moddate": {"$gte": minTime, "$lte": maxTime}}},
+            {"$match": {"moddate": {"$gte": _convert_to_datetime(minTime),
```

Comment:
ci_workspace.workspaceObjects.json has int date. This logic conflicts with test data.